src/components/App/App.js

```diff
@@ -1,61 +1,72 @@
 import React from 'react'
+import { gsap, Back } from 'gsap'
 import Header from '../Header/Header'
-import Preview from '../Preview/Preview'
 import Movies from '../Movies/Movies'
+import Preview from '../Preview/Preview'
+import apiCalls from '../../apiData/apiCalls'
 import MovieInfo from '../MovieInfo/MovieInfo'
-
-import apiCalls from '../../apiCalls'
+import cleanApiData from '../../apiData/utilities'
 import './App.css'
-import { gsap, Back } from 'gsap'
+
 class App extends React.Component {
   constructor() {
     super()
     this.state = {
       movies: '',
-      poster: '',
+      movie: '',
       video: '',
       error: '',
       effect: gsap.timeline(),
       isActive: true
     }
   }
 
-  displayPoster = id => {
-    apiCalls.fetchAMovie(id)
-      .then(data => this.setState({poster: data.movie}))
+  displayAMovie = id => {
```

Comment:
This doesn't display the movie, it fetches it. I'd watch out for these naming conventions in the future